packages/elements/src/autosuggest/__demo__/index.html

```diff
@@ -665,66 +667,21 @@
       </script>
     </demo-block>
 
-    <demo-block layout="normal" header="Inside Shadow DOM" tags="Shadow DOM,Grid">
-      <emerald-grid id="grid" style="height: 200px"></emerald-grid>
+    <demo-block layout="normal" header="Inside Shadow DOM" tags="Shadow DOM">
+      <div id="search-suggestion"></div>
       <script>
-        (function () {
-          const grid = document.getElementById('grid');
-          grid.config = {
-            rowHighlight: true,
-            rowSelection: true,
-            columns: [
-              {
-                id: 'suggest',
-                title: 'Suggest',
-                field: 'suggest',
-                formatter: {
-                  render: function () {},
-                  bind: function (rowIndex, columnIndex, value, cell, columnDef, dataRow, dataTable) {
-                    let wrapper = cell.getContent();
-                    if (!wrapper) {
-                      wrapper = document.createElement('div');
-                      const input = document.createElement('ef-text-field');
-                      input.style.height = '32px';
-                      input.style.margin = '0';
-                      input.placeholder = 'Start typing...';
-                      wrapper.appendChild(input);
-
-                      const autoSuggest = document.createElement('ef-autosuggest');
-                      autoSuggest.attach = input;
-                      autoSuggest.addEventListener(
-                        'suggestions-fetch-requested',
-                        ({ target, detail: { query } }) => {
-                          window.filterData(query, { minLength: 0 }).then(({ data }) => {
-                            target.suggestions = data;
-                          });
-                        }
-                      );
-                      wrapper.appendChild(autoSuggest);
-
-                      wrapper.input = input;
-                      wrapper.autoSuggest = autoSuggest;
-                    }
-                    cell.setContent(wrapper);
-                    wrapper.input.value = value;
-                  }
-                }
-              }
-            ],
-            dataModel: {
-              fields: ['suggest'],
-              format: 'rows',
-              data: [
-                {
-                  suggest: ''
-                },
-                {
-                  suggest: 'Cornelius Martin'
-                }
-              ]
-            }
-          };
-        })();
+        const wrapper = document.getElementById('search-suggestion');
+        let searchField = document.createElement('ef-search-field');
```

Comment:
let's replace `let` with `const` if there is no reassignment.

Reply:
Updated!